sdcm/cluster.py

```diff
@@ -2949,6 +2961,15 @@ def tags(self) -> Dict[str, str]:
                 "NodeType": str(self.node_type),
                 "keep_action": "terminate" if action == "destroy" else "", }
 
+    @property
+    def all_nodes_scylla_shards(self):
```

Comment:
I don't understand why do we need this? you can iterate over the nodes list and get the number of shards...

Reply:
add a comment in the code

Reply:
use `BaseCluster.dead_nodes_ip_address_list`, extend if needed

Reply:
done